src/test/run-pass-fulldeps/qquote.rs

```diff
@@ -9,59 +9,56 @@
 // except according to those terms.
 
 // ignore-cross-compile
-// ignore-pretty
 
 #![feature(quote, rustc_private)]
 
 extern crate syntax;
 
-use syntax::ast;
-use syntax::codemap;
-use syntax::parse;
-use syntax::print::pprust;
+use syntax::codemap::DUMMY_SP;
+use syntax::print::pprust::*;
 
-trait FakeExtCtxt {
```

Comment:
I'm not sure what you mean, but it looks to me like the main function is effectively unchanged, just rearranged.